server/src/main/java/org/opensearch/index/translog/transfer/BaseTranslogTransferManager.java

```diff
@@ -46,14 +48,13 @@
 import java.util.stream.Collectors;
 
 import static org.opensearch.index.translog.transfer.FileSnapshot.TransferFileSnapshot;
-import static org.opensearch.index.translog.transfer.FileSnapshot.TranslogFileSnapshot;
 
 /**
  * The class responsible for orchestrating the transfer of a {@link TransferSnapshot} via a {@link TransferService}
  *
  * @opensearch.internal
  */
-public class TranslogTransferManager {
+public abstract class BaseTranslogTransferManager {
```

Comment:
As i think more, it is okay to have TranslogTransferManager name itself since we have descriptive names in the implementations.